ui/app/src/components/PreviewConcierge/PreviewConcierge.tsx

```diff
@@ -550,12 +552,20 @@ export function PreviewConcierge(props: PropsWithChildren<{}>) {
         authoringBase,
         formatMessage,
         modelIdByPath,
-        enqueueSnackbar
+        enqueueSnackbar,
+        env
       } = upToDateRefs.current;
       const { type, payload } = action;
       switch (type) {
         case guestSiteLoad.type:
         case guestCheckIn.type:
+          const { version: guestVersion } = payload;
+          const studioVersion = env.version;
+
+          if (type === guestCheckIn.type && guestVersion?.substr(0, 5) !== studioVersion) {
+            enqueueSnackbar(formatMessage(guestMessages.outdatedExpBuilderVersion));
```

Comment:
Just thought this would be too intrusive as it would snack on every page navigation, refresh, edit, etc. Show it once per day.